tests/ScrumMasterTest/Unit/Concerns/JiraApiResource.php

```diff
@@ -35,7 +35,7 @@ private function createAJiraIssueAsArray(
         return [
             'key' => $key,
             'fields' => [
-                Tickets::FIELD_STORY_POINTS => '5.0',
+                (new Tickets('customfield_10005'))->getFieldStoryPoints() => '5.0',
```

Comment:
With this we got no improvement at all. The idea is that we should be able to define this string from the outside. This key and maybe more(?). I think we should be able to pass/define it from the main entry point.